app/graphql/types/query_type.rb

```diff
@@ -50,6 +58,17 @@ def digital_object(id:)
       digital_object
     end
 
+    def vocabulary(string_key:)
+      vocabulary = Vocabulary.find_by!(string_key: string_key)
+      ability.authorize!(:read, vocabulary)
+      vocabulary
+    end
+
+    def vocabularies(_arguments)
```

Comment:
Why can't this unused argument be removed?  Something to do with the pagination extension?

Reply:
Yea, in the #resolve function for the extension i have to pass  the arguments to the yield method. The vocabularies method, in this case, must take those results. I don't think there's a way around it.

Reply:
Oh okay.  Makes sense!